via/static/scripts/video_player/utils/api.ts

```diff
@@ -57,7 +68,15 @@ export async function callAPI<T = unknown>(api: APIMethod): Promise<T> {
   const resultData = await result.json().catch(() => null);
 
   if (result.status >= 400 && result.status < 600) {
-    throw new APIError(result.status, resultData);
+    let error;
+    if (
+      resultData &&
+      Array.isArray(resultData.errors) &&
+      resultData.errors.length > 0
+    ) {
+      error = resultData.errors[0];
```

Comment:
The JSON:API format allows for multiple errors, but we currently only report the first one on the frontend. I'm not aware of a scenario where we'd expect the backend to report multiple yet.